bin/cron/build_contact_rollups

```diff
@@ -40,6 +49,90 @@ def main
 
   # Any other failures along the way will be unrescued exceptions that
   # go to HoneyBadger
+ensure
+  delete_database_clone
+end
+
+# Delete database cluster clone if it exists.
+def delete_database_clone
+  rds_client = Aws::RDS::Client.new
+  begin
+    existing_cluster_response = rds_client.describe_db_clusters({db_cluster_identifier: DATABASE_CLUSTER_CLONE_ID})
+    ChatClient.message(CHAT_CHANNEL, "Cluster clone exists and is in state - #{existing_cluster_response.db_clusters.first.status}.  Deleting cluster and its instances.")
+    existing_cluster_response.db_clusters.first.db_cluster_members.each do |instance|
+      ChatClient.message(CHAT_CHANNEL, "Deleting clone instance - #{instance.db_instance_identifier}")
+      rds_client.delete_db_instance(
+        {
+          db_instance_identifier: instance.db_instance_identifier,
+          skip_final_snapshot: true,
+        }
+      )
+      rds_client.wait_until(:db_instance_deleted, {db_instance_identifier: instance.db_instance_identifier}, {max_attempts: 20, delay: 60})
+    end
+    ChatClient.message(CHAT_CHANNEL, "Deleting clone cluster - #{DATABASE_CLUSTER_CLONE_ID}")
+    rds_client.delete_db_cluster(
+      {
+        db_cluster_identifier: DATABASE_CLUSTER_CLONE_ID,
+        skip_final_snapshot: true,
+      }
+    )
+    wait_until_db_cluster_deleted(DATABASE_CLUSTER_CLONE_ID, 10, 60)
+  rescue Aws::RDS::Errors::DBClusterNotFoundFault => error
```

Comment:
Nitpick - should reduce the code in begin block to only the code this rescue block is intended for (the  `describe_db_clusters` call).

Reply:
I was intentionally using the Error to break out of the loop.  How does this change look?